### ISSUE TITLE
Deadlock in Azkaban 2.1 Web Server

### ISSUE THREAD

**Issue by gitfrederic** [azkaban/azkaban#33]
When accessing the Azkaban web server (after it successfully scheduled/executed a job), the web server deadlocked.  The JVM stack trace (after sending SIGQUIT) is:

Full thread dump OpenJDK 64-Bit Server VM (23.7-b01 mixed mode):

"Thread-10" prio=10 tid=0x00007f95d4001800 nid=0x2738 waiting on condition [0x00007f9680228000]
   java.lang.Thread.State: TIMED_WAITING (sleeping)
	at java.lang.Thread.sleep(Native Method)
	at org.mortbay.thread.QueuedThreadPool.doStop(QueuedThreadPool.java:435)
	at org.mortbay.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:76)
	- locked <0x00000007e6036748> (a java.lang.Object)
	at org.mortbay.jetty.Server.doStop(Server.java:291)
	at org.mortbay.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:76)
	- locked <0x00000007e5590298> (a java.lang.Object)
	at azkaban.webapp.AzkabanWebServer$1.run(AzkabanWebServer.java:465)

"SIGTERM handler" daemon prio=10 tid=0x00007f9654001000 nid=0x2737 in Object.wait() [0x00007f968042a000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1258)
	- locked <0x00000007556ebeb8> (a azkaban.webapp.AzkabanWebServer$1)
	at java.lang.Thread.join(Thread.java:1332)
	at java.lang.ApplicationShutdownHooks.runHooks(ApplicationShutdownHooks.java:106)
	at java.lang.ApplicationShutdownHooks$1.run(ApplicationShutdownHooks.java:46)
	at java.lang.Shutdown.runHooks(Shutdown.java:123)
	at java.lang.Shutdown.sequence(Shutdown.java:167)
	at java.lang.Shutdown.exit(Shutdown.java:212)
	- locked <0x00000007556ec008> (a java.lang.Class for java.lang.Shutdown)
	at java.lang.Terminator$1.handle(Terminator.java:52)
	at sun.misc.Signal$1.run(Signal.java:212)
	at java.lang.Thread.run(Thread.java:722)

"2102544801@qtp-1369161839-45" prio=10 tid=0x00007f9618033000 nid=0x2546 waiting for monitor entry [0x00007f9680b30000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"162434782@qtp-1369161839-44" prio=10 tid=0x00007f9618032800 nid=0x2544 waiting for monitor entry [0x00007f968062b000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1981941886@qtp-1369161839-43" prio=10 tid=0x00007f9618057000 nid=0x24cf waiting for monitor entry [0x00007f9680d32000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1662427104@qtp-1369161839-42" prio=10 tid=0x00007f9618056000 nid=0x24c6 waiting for monitor entry [0x00007f968092e000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1685440007@qtp-1369161839-41" prio=10 tid=0x00007f9618021000 nid=0x239b waiting for monitor entry [0x00007f9680e33000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"313138637@qtp-1369161839-40" prio=10 tid=0x00007f9618023000 nid=0x2398 waiting for monitor entry [0x00007f9680a2f000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1891074395@qtp-1369161839-39" prio=10 tid=0x00007f9618053800 nid=0x2396 waiting for monitor entry [0x00007f968082d000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1961647566@qtp-1369161839-36" prio=10 tid=0x00007f9618052800 nid=0x1ac3 waiting for monitor entry [0x00007f9682ed3000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1896617363@qtp-1369161839-35" prio=10 tid=0x00007f9618051800 nid=0x1ac2 waiting for monitor entry [0x00007f9680328000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"317606521@qtp-1369161839-33" prio=10 tid=0x00007f9618027800 nid=0x1ac0 waiting for monitor entry [0x00007f968072c000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"262131240@qtp-1369161839-30" prio=10 tid=0x00007f9618019000 nid=0x1ab8 waiting for monitor entry [0x00007f9680c31000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"32453141@qtp-1369161839-29" prio=10 tid=0x00007f9618025000 nid=0xb7d waiting for monitor entry [0x00007f968052a000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
		at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"52225937@qtp-1369161839-27" prio=10 tid=0x00007f9618055000 nid=0xb76 waiting for monitor entry [0x00007f96831d6000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"1057409309@qtp-1369161839-3" prio=10 tid=0x00007f961802a800 nid=0xdbf2 waiting for monitor entry [0x00007f9680f34000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager$ScheduleRunner.getRunnerSchedules(ScheduleManager.java:292)
	- waiting to lock <0x00000007e60f5038> (a azkaban.scheduler.ScheduleManager$ScheduleRunner)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- locked <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)

"DestroyJavaVM" prio=10 tid=0x00007f970800a800 nid=0xdba9 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Thread-6" prio=10 tid=0x00007f9708738000 nid=0xdbe1 waiting for monitor entry [0x00007f96832d8000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at azkaban.scheduler.ScheduleManager.removeSchedule(ScheduleManager.java:139)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.scheduler.ScheduleManager$ScheduleRunner.run(ScheduleManager.java:444)
	- locked <0x00000007e60f5038> (a azkaban.scheduler.ScheduleManager$ScheduleRunner)

"AzkabanWebServer-Cleaner-Thread" prio=10 tid=0x00007f9708762800 nid=0xdbdb in Object.wait() [0x00007f96834da000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <0x00000007e6125d60> (a azkaban.executor.ExecutorManager$CleanerThread)
	at azkaban.executor.ExecutorManager$CleanerThread.run(ExecutorManager.java:966)
	- locked <0x00000007e6125d60> (a azkaban.executor.ExecutorManager$CleanerThread)

"Abandoned connection cleanup thread" daemon prio=10 tid=0x00007f9708715000 nid=0xdbd5 in Object.wait() [0x00007f96836dc000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:135)
	- locked <0x00000007e6150518> (a java.lang.ref.ReferenceQueue$Lock)
	at com.mysql.jdbc.AbandonedConnectionCleanupThread.run(AbandonedConnectionCleanupThread.java:40)

"net.sf.ehcache.CacheManager@71ac4afb" daemon prio=10 tid=0x00007f97085e1000 nid=0xdbd4 in Object.wait() [0x00007f96837dd000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <0x00000007e5575470> (a java.util.TaskQueue)
	at java.util.TimerThread.mainLoop(Timer.java:552)
	- locked <0x00000007e5575470> (a java.util.TaskQueue)
	at java.util.TimerThread.run(Timer.java:505)

"RMI TCP Accept-0" daemon prio=10 tid=0x00007f97083cf000 nid=0xdbd2 runnable [0x00007f96839df000]
   java.lang.Thread.State: RUNNABLE
	at java.net.PlainSocketImpl.socketAccept(Native Method)
	at java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:398)
	at java.net.ServerSocket.implAccept(ServerSocket.java:522)
	at java.net.ServerSocket.accept(ServerSocket.java:490)
	at sun.management.jmxremote.LocalRMIServerSocketFactory$1.accept(LocalRMIServerSocketFactory.java:52)
	at sun.rmi.transport.tcp.TCPTransport$AcceptLoop.executeAcceptLoop(TCPTransport.java:387)
	at sun.rmi.transport.tcp.TCPTransport$AcceptLoop.run(TCPTransport.java:359)
	at java.lang.Thread.run(Thread.java:722)

"Service Thread" daemon prio=10 tid=0x00007f970814e000 nid=0xdbd1 runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread1" daemon prio=10 tid=0x00007f970814c000 nid=0xdbd0 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" daemon prio=10 tid=0x00007f9708149000 nid=0xdbcf waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" daemon prio=10 tid=0x00007f9708147000 nid=0xdbce waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" daemon prio=10 tid=0x00007f97080f0800 nid=0xdbcd in Object.wait() [0x00007f9689fde000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:135)
	- locked <0x00000007e60e0c30> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:151)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:189)

"Reference Handler" daemon prio=10 tid=0x00007f97080ee000 nid=0xdbcc in Object.wait() [0x00007f968a0df000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:503)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:133)
	- locked <0x00000007e6030298> (a java.lang.ref.Reference$Lock)

"VM Thread" prio=10 tid=0x00007f97080e6000 nid=0xdbcb runnable 

"GC task thread#0 (ParallelGC)" prio=10 tid=0x00007f9708018800 nid=0xdbaa runnable 

"GC task thread#1 (ParallelGC)" prio=10 tid=0x00007f970801a800 nid=0xdbab runnable 

"GC task thread#2 (ParallelGC)" prio=10 tid=0x00007f970801c000 nid=0xdbac runnable 

"GC task thread#3 (ParallelGC)" prio=10 tid=0x00007f970801e000 nid=0xdbad runnable 

"GC task thread#4 (ParallelGC)" prio=10 tid=0x00007f9708020000 nid=0xdbae runnable 

"GC task thread#5 (ParallelGC)" prio=10 tid=0x00007f9708021800 nid=0xdbaf runnable 

"GC task thread#6 (ParallelGC)" prio=10 tid=0x00007f9708023800 nid=0xdbb0 runnable 

"GC task thread#7 (ParallelGC)" prio=10 tid=0x00007f9708025800 nid=0xdbb1 runnable 

"GC task thread#8 (ParallelGC)" prio=10 tid=0x00007f9708027000 nid=0xdbb2 runnable 

"GC task thread#9 (ParallelGC)" prio=10 tid=0x00007f9708029000 nid=0xdbb3 runnable 

"GC task thread#10 (ParallelGC)" prio=10 tid=0x00007f970802b000 nid=0xdbb4 runnable 

"GC task thread#11 (ParallelGC)" prio=10 tid=0x00007f970802c800 nid=0xdbb5 runnable 

"GC task thread#12 (ParallelGC)" prio=10 tid=0x00007f970802e800 nid=0xdbb6 runnable 

"GC task thread#13 (ParallelGC)" prio=10 tid=0x00007f9708030800 nid=0xdbb7 runnable 

"GC task thread#14 (ParallelGC)" prio=10 tid=0x00007f9708032000 nid=0xdbb8 runnable 

"GC task thread#15 (ParallelGC)" prio=10 tid=0x00007f9708034000 nid=0xdbb9 runnable 

"GC task thread#16 (ParallelGC)" prio=10 tid=0x00007f9708036000 nid=0xdbba runnable 

"GC task thread#17 (ParallelGC)" prio=10 tid=0x00007f9708038000 nid=0xdbbb runnable 

"GC task thread#18 (ParallelGC)" prio=10 tid=0x00007f9708039800 nid=0xdbbc runnable 

"GC task thread#19 (ParallelGC)" prio=10 tid=0x00007f970803b800 nid=0xdbbd runnable 

"GC task thread#20 (ParallelGC)" prio=10 tid=0x00007f970803d800 nid=0xdbbe runnable 

"GC task thread#21 (ParallelGC)" prio=10 tid=0x00007f970803f000 nid=0xdbbf runnable 

"GC task thread#22 (ParallelGC)" prio=10 tid=0x00007f9708041000 nid=0xdbc0 runnable 

"GC task thread#23 (ParallelGC)" prio=10 tid=0x00007f9708043000 nid=0xdbc1 runnable 

"GC task thread#24 (ParallelGC)" prio=10 tid=0x00007f9708044800 nid=0xdbc2 runnable 

"GC task thread#25 (ParallelGC)" prio=10 tid=0x00007f9708046800 nid=0xdbc3 runnable 

"GC task thread#26 (ParallelGC)" prio=10 tid=0x00007f9708048800 nid=0xdbc4 runnable 

"GC task thread#27 (ParallelGC)" prio=10 tid=0x00007f970804a000 nid=0xdbc5 runnable 

"GC task thread#28 (ParallelGC)" prio=10 tid=0x00007f970804c000 nid=0xdbc6 runnable 

"GC task thread#29 (ParallelGC)" prio=10 tid=0x00007f970804e000 nid=0xdbc7 runnable 

"GC task thread#30 (ParallelGC)" prio=10 tid=0x00007f970804f800 nid=0xdbc8 runnable 

"GC task thread#31 (ParallelGC)" prio=10 tid=0x00007f9708051800 nid=0xdbc9 runnable 

"GC task thread#32 (ParallelGC)" prio=10 tid=0x00007f9708053800 nid=0xdbca runnable 

"VM Periodic Task Thread" prio=10 tid=0x00007f97083f7800 nid=0xdbd3 waiting on condition 

JNI global references: 267

Found one Java-level deadlock:
=============================
"2102544801@qtp-1369161839-45":
  waiting to lock monitor 0x00007f9604002078 (object 0x00000007e60ec4f8, a azkaban.scheduler.ScheduleManager),
  which is held by "1057409309@qtp-1369161839-3"
"1057409309@qtp-1369161839-3":
  waiting to lock monitor 0x00007f9630005ef0 (object 0x00000007e60f5038, a azkaban.scheduler.ScheduleManager$ScheduleRunner),
  which is held by "Thread-6"
"Thread-6":
  waiting to lock monitor 0x00007f9604002078 (object 0x00000007e60ec4f8, a azkaban.scheduler.ScheduleManager),
  which is held by "1057409309@qtp-1369161839-3"

Java stack information for the threads listed above:
===================================================
"2102544801@qtp-1369161839-45":
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)
"1057409309@qtp-1369161839-3":
	at azkaban.scheduler.ScheduleManager$ScheduleRunner.getRunnerSchedules(ScheduleManager.java:292)
	- waiting to lock <0x00000007e60f5038> (a azkaban.scheduler.ScheduleManager$ScheduleRunner)
	at azkaban.scheduler.ScheduleManager.getSchedules(ScheduleManager.java:120)
	- locked <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.webapp.servlet.ScheduleServlet.handleGetAllSchedules(ScheduleServlet.java:292)
	at azkaban.webapp.servlet.ScheduleServlet.handleGet(ScheduleServlet.java:80)
	at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:77)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at org.mortbay.jetty.security.SslSocketConnector$SslConnection.run(SslSocketConnector.java:713)
	at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)
"Thread-6":
	at azkaban.scheduler.ScheduleManager.removeSchedule(ScheduleManager.java:139)
	- waiting to lock <0x00000007e60ec4f8> (a azkaban.scheduler.ScheduleManager)
	at azkaban.scheduler.ScheduleManager$ScheduleRunner.run(ScheduleManager.java:444)
	- locked <0x00000007e60f5038> (a azkaban.scheduler.ScheduleManager$ScheduleRunner)

Found 1 deadlock.

Heap
 PSYoungGen      total 611648K, used 376044K [0x00000007c0000000, 0x00000007eaaa0000, 0x0000000800000000)
  eden space 524288K, 55% used [0x00000007c0000000,0x00000007d19ebd90,0x00000007e0000000)
  from space 87360K, 99% used [0x00000007e5550000,0x00000007eaa9f430,0x00000007eaaa0000)
  to   space 87360K, 0% used [0x00000007e0000000,0x00000007e0000000,0x00000007e5550000)
 ParOldGen       total 1398144K, used 490971K [0x0000000740000000, 0x0000000795560000, 0x00000007c0000000)
  object space 1398144K, 35% used [0x0000000740000000,0x000000075df76d68,0x0000000795560000)
 PSPermGen       total 27008K, used 26818K [0x0000000735a00000, 0x0000000737460000, 0x0000000740000000)
  object space 27008K, 99% used [0x0000000735a00000,0x0000000737430a78,0x0000000737460000)

**Comment by davidzchen** [comment]
Given how old this issue is, it is most likely an obsolete issue. Please open a new issue if you can reproduce.
